packages/tokens/src/style-dictionary/config.ts

```diff
@@ -47,7 +47,7 @@ export function getStyledSystemTokensConfig(mode: "light" | "dark"): Config {
                 },
                 "files": [
                     {
-                        "destination": `tokens/generated/${isLightMode ? "light-tokens" : "dark-semantic-tokens" }.ts`,
+                        "destination": `${isLightMode ? "light-tokens" : "dark-semantic-tokens" }.ts`,
```

Comment:
curious as to why light tokens are not named light-semantic-tokens

Reply:
good point thanks